projects/aca-shared/rules/src/app.rules.spec.ts

```diff
@@ -1207,58 +1032,35 @@ describe('app.evaluators', () => {
     });
   });
 
-  describe('isTrashcanItemSelected', () => {
-    it('should return false when user is not in trashcan', () => {
-      context.navigation.url = '/personal-files/test';
-      expect(app.isTrashcanItemSelected(context)).toBeFalse();
-    });
-
-    it('should return false when nothing is selected', () => {
-      context.navigation.url = '/trashcan/test';
-      context.selection.isEmpty = true;
-      expect(app.isTrashcanItemSelected(context)).toBeFalse();
-    });
-
-    it('should return true when user is in trashcan and node is selected', () => {
-      context.navigation.url = '/trashcan/test';
-      context.selection.isEmpty = false;
-      expect(app.isTrashcanItemSelected(context)).toBeTrue();
-    });
-  });
-
-  describe('canViewFile', () => {
-    it('should return false when user is in trashcan', () => {
-      context.navigation.url = '/trashcan/test';
-      expect(app.canViewFile(context)).toBeFalse();
-    });
-
-    it('should return false when nothing is selected', () => {
-      context.navigation.url = '/personal-files/test';
-      context.selection.file = null;
-      expect(app.canViewFile(context)).toBeFalse();
+  describe('canToggleFileLock', () => {
+    beforeEach(() => {
+      context.profile = {} as any;
```

Comment:
Instead of any please use as ProfileState, works as well and it's better type: 
<img width="483" alt="image" src="https://github.com/user-attachments/assets/608f95c9-7ebc-4f69-8dab-3664db96a0be">

Reply:
Replaced any  with ProfileState